core/subsurface-qt/diveobjecthelper.cpp

```diff
@@ -249,6 +249,28 @@ QStringList getFullCylinderList()
 	return cylinders;
 }
 
+QString formatSalinity(const dive *d)
+{
+	int salinity = get_dive_salinity(d);
+	if (!salinity)
+		return QString();
+        return QString("%L1g/ℓ").arg(salinity / 10.0);
```

Comment:
i think this needs to be just

```
return gettextFromC::tr("%1g/ℓ").arg(salinity / 10.0);
```

the g/l part needs localization too and i don't see `%L` elsewhere in the codebase according to github search.

Reply:
What @bstoeger was saying is that ``%L1`` should magically handle the commas/periods. The ``g/l`` doesn't get localized because it's _always_ g/l - we don't have imperial units for density. Also - ``git grep '%L1'`` shows tons of hits, so I think it's just something github search can't search for.

Reply:
but in my language it's `%1г/л` and we do localize strings like that elsewhere.

Reply:
Gotcha ... and will do!

Reply:
> so if you are finding %L then i guess it should be:

Yes, both should be done as these are orthogonal issues [you can set language and decimal separators independently]. An alternative might be:
```
QStringLiteral("%L1%2").arg(salinity / 10.0, gettextFromC::tr("g/ℓ"))
```

Reply:
Your version is cleaner - I've updated to use that instead.